style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --header-mobile-color: #3a4a42;
+  --header-desktop-color: #344563;
+  --headline-color-1: #172b4d;
+  --headline-color-2: #36b37e;
+  --background-color: #fff;
+}
+
+/* Reset the default browser styles */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+/* Global styles */
+body {
+  background: var(--background-color);
+}
+
+/* Navbar */
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 48px;
+}
+
+/* Hide the nav bar on mobile */
+.navbar ul {
+  display: none;
+}
+
+/* Hamburger menu */
+.hamburger_toggle {
+  display: block;
+  cursor: pointer;
+  margin-right: 1rem;
+  margin-top: 0.75rem;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 2rem;
+  height: 1.75rem;
+  background: var(--background-color);
+  border: none;
+}
+
+.navbar li {
+  margin: 0 1rem;
+}
+
+.navbar li a {
+  color: var(--header-mobile-color);
+}
+
+/* Section headline */
+.headline {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+}
+
+/* Social icons */
+.headline .headline-social {
+  display: flex;
+}
+
+/* Scroll button */
+.scroll-button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* mobile screen size */
+@media screen and (min-width: 375px) {
+  nav {
+    height: 48px;
+    top: 44px;
+  }
+
+  /* Show the hamburger menu */
+  .hamburger_toggle {
+    display: block;
+  }
+
+  .navbar ul {
+    display: none;
+  }
+
+  /* Section headline */
+  .headline {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    height: 100vh;
+    background: url('./assets/background_header_mob.svg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    top: 55.17px;
+    left: 0;
+    bottom: 0;
+    overflow: hidden;
+  }
```

Comment:
- [ ] **[OPTIONAL]** I suggest not overusing the `position: absolute` property since it gets elements out of the normal document flow. It is most used for floating elements, it would be great if you can find a way of not using it in the entire section! 🤓 🔧

Reply:
Alright thanks for the advice @anagudelogu